includes/class-wc-stripe-payment-method-configurations.php

```diff
@@ -121,9 +121,10 @@ private static function get_payment_method_configuration_from_cache( $use_fallba
 	 */
 	public static function clear_payment_method_configuration_cache() {
 		self::$primary_configuration = null;
-		$cache_key                   = WC_Stripe_Mode::is_test() ? self::TEST_MODE_CONFIGURATION_CACHE_TRANSIENT_KEY : self::LIVE_MODE_CONFIGURATION_CACHE_TRANSIENT_KEY;
-		delete_transient( $cache_key );
+		$cache_key                   = WC_Stripe_Mode::is_test() ? self::TEST_MODE_CONFIGURATION_CACHE_KEY : self::LIVE_MODE_CONFIGURATION_CACHE_KEY;
+		WC_Stripe_Database_Cache::delete( $cache_key );
 		delete_option( $cache_key );
```

Comment:
```suggestion
```

I think this is redundant -- `delete_option( $cache_key )` is already performed by the preceding line.